support-frontend/assets/components/StripePaymentRequestButton/StripePaymentRequestButton.tsx

```diff
@@ -65,16 +60,33 @@ import {
 } from 'pages/contributions-landing/contributionsLandingActions';
 import type { Action } from 'pages/contributions-landing/contributionsLandingActions';
 import type { State } from 'pages/contributions-landing/contributionsLandingReducer';
-import { trackComponentEvents } from '../../../../helpers/tracking/ophan';
+import { trackComponentEvents } from '../../helpers/tracking/ophan';
 
 // ----- Types -----//
 
+type PrbType = 'APPLE_PAY' | 'GOOGLE_PAY' | 'PAY_NOW' | 'NONE';
+
+export interface RenderPrbInput {
+	type: PrbType;
+	paymentRequest: PaymentRequest;
+	paymentRequestError: ErrorReason | null;
+	onStripePrbClick: (
+		event: StripePaymentRequestButtonElementClickEvent,
+	) => void;
+	onCustomPrbClick: (
+		event: StripePaymentRequestButtonElementClickEvent,
+	) => void;
+}
+
+export type RenderPrb = (input: RenderPrbInput) => JSX.Element;
```

Comment:
I feel this is maybe slightly ugly, but I added this type & made the input to the render prop a single object so consumers can selectively destructure what they need

Reply:
I'm not a big fan of the Prb acronym, I think it's confusing for people unfamiliar with the code base - could we call it `RenderPaymentRequestButton`? It's a bit verbose but I think probably worth it for the clarity

Reply:
Yeah that's a good suggestion. I'll comb through and replace uses of `Prb`